packages/fluentui/react-bindings/src/hooks/useStyles.ts

```diff
@@ -36,7 +36,8 @@ const defaultContext: StylesContextValue<{ renderRule: RendererRenderRule }> = {
   performance: {
     enableSanitizeCssPlugin: process.env.NODE_ENV !== 'production',
     enableStylesCaching: true,
-    enableVariablesCaching: true
+    enableVariablesCaching: true,
+    enableHardVariablesCaching: false
```

Comment:
Why not `enableBooleanVariablesCaching`?

Reply:
Renamed, it's definitely sounds better